esp/scm/ws_dfs.ecm

```diff
@@ -0,0 +1,64 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2012 HPCC Systems®.
```

Comment:
(c) date.  Likely to be true for other files as well.

Reply:
will fix in new files